packages/TesterApp/android/gradle.properties

```diff
@@ -34,7 +34,7 @@ reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
 # your application. You should enable this flag either if you want
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
-newArchEnabled=false
+newArchEnabled=true
```

Comment:
@RafikiTiki might be an unneeded change, we got to decide whether we run TesterApp in newarch by default or not

Reply:
Let's use newArch to be consistent with iOS